src/sparseml/pytorch/sparsification/pruning/modifier_pruning_base.py

```diff
@@ -174,8 +174,12 @@ def sparsification_types(self) -> List[SparsificationTypes]:
         return [SparsificationTypes.pruning]
 
     @abstractmethod
-    def _get_mask_creator(self) -> PruningMaskCreator:
+    def _get_mask_creator(
```

Comment:
nit: private method should go after the public method :)

Reply:
good catch, going to keep this one at top for now since it's an abstract method and want it to be called out